app/views/includes/printBanner.scala.html

```diff
@@ -14,13 +14,21 @@
  * limitations under the License.
  *@
 
-@(preHeadingText: String)(implicit m: Messages)
+@(preHeadingText: String, userName: String = "")(implicit m: Messages)
+
+@printDisplayName = {
```

Comment:
u can also do  
```@userName.getOrElse(preHeadingText)```

Reply:
if i am using a .getOrElse do i need to make line 17 @(preHeadingText: String, userName: Option[String] = None)... ?